src/bp/ui-studio/src/web/components/Layout/BottomPanel/Logs/index.tsx

```diff
@@ -61,8 +61,11 @@ class BottomPanel extends React.Component<Props, State> {
   }
 
   setupListener = () => {
-    // @ts-ignore
-    EventBus.default.on(`logs::${window.BOT_ID}`, ({ id, level, message, args }) => {
+    EventBus.default.onAny((name, { level, message, args }) => {
+      if (!name || typeof name !== 'string' || ![`logs::${window.BOT_ID}`, 'logs::*'].includes(name)) {
```

Comment:
Can you also either add a comment or move the condition into a variable so that it reflects better what it does. 

I find it a little harder than it should be to deduce that this `![`logs::${window.BOT_ID}`, 'logs::*'].includes(name)` means discard all the logs that are not global or associated with a certain bot.

Reply:
Done!